tests/integration/golang/mlflow/run/create_test.go

```diff
@@ -128,27 +236,62 @@ func (s *CreateRunTestSuite) Test_Error() {
 			),
 		},
 		{
-			name: "CreateRunWithNotExistingExperiment",
+			name:      "CreateRunWithNotExistingNamespaceAndExistingExperimentID",
+			namespace: "not_existing_namespace",
 			request: request.CreateRunRequest{
-				ExperimentID: "1",
+				ExperimentID: fmt.Sprintf("%d", *experiment.ID),
 			},
 			error: api.NewResourceDoesNotExistError(
-				`unable to find experiment with id '1': error getting experiment by id: 1: record not found`,
+				`unable to find namespace with code: not_existing_namespace`,
+			),
```

Comment:
probably this test tests `namespace` functionality and `middleware` itself. We don't need to have this test here It will be included into my PR related to `namespace` integration tests.

Reply:
I was just adapting the existing test for namespace inputs

Reply:
I'm vetoing the scope creep here.